routes/auth.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const router = express.Router();
```

Comment:
```suggestion
const authRouter = express.Router();
```
authRouter 더 명시적일 것 같아요